src/std/net/s3/sigv4-test.ss

```diff
@@ -0,0 +1,55 @@
+(export sigv4-test)
+
+(import (only-in :std/test test-suite test-case check)
+        (only-in :std/crypto/digest sha256)
+        (only-in :std/text/hex hex-encode)
+        (only-in "./sigv4" aws4-canonical-request aws4-sign aws4-auth))
```

Comment:
please dont use the quoted form, that is a source import.

Just `./sigv4` is fine, it will do the right thing.

Reply:
Ah, thank you for pointing this out. I have updated the import